SLAM/landmark_matcher.py

```diff
@@ -0,0 +1,14 @@
+import SLAM.landmark_extractor
+import SLAM.position_evaluator
+
+total_landmarks = []
+
+def match_landmarks(landmarks, stored_landmarks):
+    #compare vectors
+    for landmark in landmarks:
+        for existing_landmark in stored_landmarks:
+            if landmark.AB==existing_landmark.AB and landmark.BC==landmark.BC:
+                SLAM.position_evaluator(landmark)
+            else:
+                total_landmarks.append(landmark)
+    return total_landmarks
```

Comment:
If this function calls the postion_evaluator, but only returns the total_landmarks - how can we forward the new robot position to the pointcloud_generator or sender?